charts/kor/values.yaml

```diff
@@ -0,0 +1,26 @@
+namespace: kor
```

Comment:
Can you remove the namespace from the values? (let the user set it through the -n flag)

Reply:
values file is just a default value, the user can modify it by passing it through the helm command line on installation/upgrade time.

If I remove this default value and leave only the flag -n on helm command line it would place the helm chart creation on kor namespace but subjects ClusterRoleBinding need the namespace value to be deployed.
I removed it locally and tested and got this error:

```
Error: UPGRADE FAILED: cannot patch "kor-read-resources-clusterrolebinding" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "kor-read-resources-clusterrolebinding" is invalid: subjects[0].namespace: Required value
```

If I remove it the user would need to also pass the set flag `--set namespace=kor` on helm command line:

```
helm upgrade -i kor --namespace kor --create-namespace --set namespace=kor --set cronJob.slackChannel=<slack-channel> --set cronJob.slackAuthToken=<user-token> ./charts/kor
```

What do you think is best? I think it's redundant to have both `--set namespace=kor` and `--namespace kor` in the same command. Even though the user can still do it with the default value set to `namespace: kor`.